tests/test_spatial_combine_transforms.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import unittest
+
+import numpy as np
+import torch
+from parameterized import parameterized
+
+from monai.data.meta_tensor import MetaTensor
+from monai.transforms import (
+    Orientation,
+    Orientationd,
+    RandFlip,
+    RandFlipd,
+    Randomizable,
+    RandRotate,
+    RandRotated,
+    RandSpatialCrop,
+    RandZoom,
+    RandZoomd,
+    Resize,
+    Resized,
+    Spacing,
+    Spacingd,
+)
+from monai.transforms.transform import MapTransform
+from tests.utils import assert_allclose
+
+TEST_2D = [
+    [
+        (2, 64, 64),
+        [
+            (Spacing, {"pixdim": (1.2, 1.5), "padding_mode": "zeros", "dtype": torch.float32}),
+            (Orientation, {"axcodes": "RA"}),
+            (Resize, {"spatial_size": (48, 48), "mode": "bilinear"}),
+            (RandSpatialCrop, {"roi_size": (32, 32)}),
+            # TODO: the following transform should also work?
+            # (RandAffine, {"prob": 0.9, "rotate_range": (np.pi / 2,), "shear_range": [1, 2], "translate_range": [2, 1], "mode": "bilinear"}),
+            (RandFlip, {"prob": 0.9}),
+            (RandRotate, {"prob": 0.9, "range_x": np.pi / 4}),
+            (RandZoom, {"prob": 0.9, "mode": "bilinear", "keep_size": False, "align_corners": False}),
+        ],
+    ],
+    [
+        (2, 64, 64),
+        [
+            # TODO: the following transform should also work?
+            # (RandSpatialCropd, {"roi_size": (32, 32), "keys": "img"}),
```

Comment:
put crop after orientation works, but at the beginning failed.

Reply:
The issue was in the `spacing`. Also see this comment: https://github.com/Project-MONAI/MONAI/pull/5860#discussion_r1133613414

Reply:
thanks, I put a patch here https://github.com/Project-MONAI/MONAI/pull/5860/commits/cc89d7a7f7592d5337e6d697fc86d07c36f6fa19

Reply:
cherry-picked that change to this branch, so I mark this as resolved.